src/callbacks_wandb.py

```diff
@@ -235,4 +235,3 @@ def on_validation_end(
                 axs[1, i].axis("off")
 
             self.logger.experiment.log({"Images": wandb.Image(fig)})
-            plt.close(fig)
```

Comment:
Can you undo the line removal here manually? Can't make a suggested change on a deleted line